burotel/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "indico-plugin-burotel",
+  "version": "1.0.0",
```

Comment:
I'd add `"private": true` here and get rid of all the stuff (author, license, urls, version) that's only useful when publishing a package